doc/content/the-things-stack/host/aws/ecs/changelog/index.md

```diff
@@ -8,6 +8,8 @@ All meaningful changes to templates are documented in this file.
 
 ## Unreleased
 
+## 3.32.3
```

Comment:
This is incorrect right?

Reply:
Seems correct, but I don't know why it wasn't added with `v3.32.3` https://github.com/TheThingsIndustries/lorawan-stack-aws/blob/v3.33/UPGRADING.md#3323

Unless it's incorrect in `lw-stack-aws` as well.

Reply:
We are releasing documentation for `v3.33.0` and the latest change here says it's for `v3.32.3` and that doesn't seem incorrect?

Reply:
Looking through the process the reason for this incorrectness in this release and the previous release is the fact that this PR [Release v3.33.0](https://github.com/TheThingsIndustries/lorawan-stack/pull/4492) is not merged, which is the trigger to AWS templates release action.

Turns out that's the reason why tag release and docs release wasn't done in parallel -> docs release needs this PR to be merged already and AWS release finished.

Reply:
Fixed in https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/1395/commits/1199c62cf8f00337ff842fbd8015aeaef96ce217 to match `UPGRADING.md`